keyboards/kwstudio/pisces/info.json

```
@@ -0,0 +1,127 @@
+{
+    "keyboard_name": "Pisces",
+    "manufacturer": "kws",
+    "maintainer": "kws",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
```

This is preferred to be left enabled if possible.
```suggestion
        "mousekey": true,
```